### PR TITLE
New calc

### PR DESCRIPTION
**Pull request by reidjs**

### CODE REVIEW

File: src/pages/calculator.js [hunk 1]
```diff
@@ -1,350 +1,346 @@
...
+            <input className="form-control" type="text" onChange={(e) => this.setCpiFromZip(e)} placeholder="Your 5 digit zip code" />
+            {this.state.town
+              && (
+              <small><strong>{this.state.town}</strong>{this.state.county
+                && <strong>, {this.state.county} County</strong>}
+              </small>
+              )}
+            <br />
+            <br />
+            <br />
```
Comment (krpeacock) [inline src/pages/calculator.js]: I mean, it's not like this is invalid but...

File: src/styles/calculator.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
+/* TODO: Convert to SCSS */
```
Comment (krpeacock) [inline src/styles/calculator.css]: You can just rename it to a .scss file without any changes if you want. Scss is a superset of CSS

Comment (reidjs) [reply src/styles/calculator.css]: oh wow, today i learned something haha

File: src/pages/calculator.js (hunk 1 above)
Comment (reidjs) [reply src/pages/calculator.js]: yeah this is hacky, should be done w/ margin IMO

Comment (sh1mmer) [reply src/pages/calculator.js]: This should totally be <p></p><p></p><p></p> like in my code

**Pull request merged**